src/mqtt_client/mod.rs

```diff
@@ -139,7 +138,7 @@ impl<'a> Command<'a> {
 ///     foo: bool,
 /// }
 ///
-/// let mut client: MqttClient<Settings, _, _, 256> = MqttClient::new(
+/// let mut client: MqttClient<_, _, _, 256, 1> = MqttClient::new(
```

Comment:
```suggestion
/// let mut client: MqttClient<_, _, _, 256, _> = MqttClient::new(
```

Or does this not work?

Reply:
Appears to not work unfortunately.